package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@datastax/astra-db-ts",
-  "version": "0.1.3",
+  "version": "0.1.4",
   "description": "Astra DB TS Client",
   "contributors": [
     "CRW (http://barnyrubble.tumblr.com/)",
```

Comment:
Not needed for the PR but would be good to clean up the contributors or remove it (ik CRW doesn't work here anymore)